clusterloader2/testing/node-throughput/windows_override.yaml

```diff
@@ -0,0 +1,3 @@
+POD_COUNT: 80
```

Comment:
out of curiosity: why do you decrease number of pods? It was 100 before.

Reply:
We were never testing windows nodes - so it's hard to say what it was before :)